assets/js/controllers/buySell.controller.js

```diff
@@ -222,7 +222,10 @@ function BuySellCtrl ($rootScope, $scope, $state, Alerts, Wallet, currency, buyS
     options: $rootScope.inMobileBuy || !$scope.canSeeSellTab
       ? ['BUY_BITCOIN', 'ORDER_HISTORY']
       : ['BUY_BITCOIN', 'SELL_BITCOIN', 'ORDER_HISTORY'],
-    select (tab) { this.selectedTab = this.selectedTab ? tab : null; }
+    select (tab) {
+      this.selectedTab = this.selectedTab ? tab : null;
+      $state.params.selectedTab = this.selectedTab;
```

Comment:
Why do you have to set `$state.params` here? If you really have to, could you use `$state.go` instead: `$state.go($state.current.name, { selectedTab: ... });`. In doing that, might be able to get rid of some unnecessary state, because we can use `$state.params.selectedTab` instead of `this.selectedTab`.